framework/helpers/BaseHtml.php

```diff
@@ -660,12 +660,20 @@ public static function fileInput($name, $value = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * The following special options are recognized:
+     *
+     * - `doubleEncode`: whether to encode HTML entities in `$value`. If `false`, HTML entities in `$value` will not
```

Comment:
Does it need to be as it references `HTML entities` which means they are already encoded.